src/OpenTracing/ISpanBuilder.cs

```diff
@@ -1,82 +1,174 @@
+        /// <seealso cref="IScopeManager" />
+        /// <seealso cref="IScope"/>
+        /// <seealso cref="StartActive(bool)"/>
+        IScope StartActive();
 
         /// <summary>
-        /// Specify a timestamp of when the span was started.
+        ///     Returns a newly started and activated <see cref="IScope" />.
+        ///     <para>
+        ///         The returned <see cref="IScope" /> supports using(). For example:
+        ///         <code>
+        /// using (IScope scope = tracer.BuildSpan("...").StartActive(false))
+        /// {
+        ///     // (Do work)
+        ///     scope.Span.SetTag( ... );  // etc, etc
+        /// }
+        /// // Span does not finish automatically when the Scope is closed as
+        /// // 'finishOnClose' is false
+        /// </code>
+        ///     </para>
+        ///     <para>
+        ///         If
+        ///         <list type="bullet">
+        ///             <item>the <see cref="ITracer" />'s <see cref="IScopeManager.Active" /> is not null, and</item>
+        ///             <item>no <b>explicit</b> references are added via <see cref="AddReference" />, and</item>
+        ///             <item><see cref="IgnoreActiveSpan" /> is not invoked,</item>
+        ///         </list>
+        ///         ... then an inferred <see cref="References.ChildOf" /> reference is created to the
+        ///         <see cref="IScopeManager.Active" /> <see cref="ISpanContext" /> when either
+        ///         <see cref="StartManual" /> or <see cref="StartActive()" /> is invoked.
+        ///     </para>
+        ///     <para>
+        ///         NOTE: <see cref="StartActive(bool)" /> is a shorthand for
+        ///         <code>tracer.MakeActive(spanBuilder.StartManual(), finishSpanOnClose)</code>
+        ///     </para>
         /// </summary>
-        /// <param name="startTimestamp">An explicit start timestamp for the span.</param>
-        ISpanBuilder WithStartTimestamp(DateTimeOffset startTimestamp);
+        /// <param name="finishSpanOnClose">Whether span sould automatically be finished when <see cref="IDisposable.Dispose"/> is called</param>
+        /// <returns>An <see cref="IScope" />, already registered via the <see cref="IScopeManager" /></returns>
+        /// <seealso cref="IScopeManager" />
+        /// <seealso cref="IScope" />
+        IScope StartActive(bool finishSpanOnClose);
 
         /// <summary>
-        /// Returns the started span.
+        ///     Like <see cref="StartActive()" />, but the returned <see cref="ISpan" /> has not been registered via the
+        ///     <see cref="IScopeManager" />.
         /// </summary>
+        /// <returns>
+        ///     The newly-started Span instance, which as *not* been automatically registered
+        ///     via the <see cref="IScopeManager" />
+        /// </returns>
+        /// <seealso cref="StartActive()" />
+        ISpan StartManual();
```

Comment:
StartManual is deprecated in Java so we don't even need it

Reply:
THANK LORDY IF WE CAN GET RID OF IT! <3 <3
That said, `StartActive` is documented to call this internally in it's implementation, so it would be necessary to keep it, would it not? I will review the latest in the javas